models/intermediate/int_amazon_selling_partner__item.sql

```diff
@@ -0,0 +1,179 @@
+with item_summary as (
+    select *
+    from {{ ref('stg_amazon_selling_partner__item_summary') }}
+),
```

Comment:
Isn't this associated with a variable? If so, we need to add a config here.

Reply:
added